galoy.yaml

```diff
@@ -130,8 +130,6 @@ accountLimits:
 spamLimits:
   memoSharingSatsThreshold: 1000
   memoSharingCentsThreshold: 50
-twoFALimits:
```

Comment:
This still exists in the `YamlSchema` type. Is the plan to remove from there as well?

Reply:
good point. removed.

yes the goal is to re-introduce 2fa in a different way. instead of attaching 2fa to payment call, it should be a separate `validate2fa` graphql call, that will give you an elevated session for the next x minutes. 

in the UI, then you would do the 2fa, and move on the validation screen next if success. then the call in the backend for the payment would be same whether it needs 2fa or not. 

I think it's a much better design than what we have attempted so far (having 2fa as part of the payments API)